src/Utils/JsLoader.php

```diff
@@ -56,15 +55,9 @@ public static function setMapOptions(array $options): void
     public static function load(App $app, string $locationUrl = null): void
     {
         if (!self::$isLoaded) {
-            if (!$locationUrl) {
-                $cdn = self::$cdn;
-                $version = self::$version;
-                $locationUrl = "{$cdn}@{$version}/public/atk-google-maps.min.js";
-            }
-
-            $app->requireJs($locationUrl);
+            $app->requireJs($locationUrl ?? self::$defaultLocationUrl);
 
-            if (!self::$apiKey) {
+            if (self::$apiKey === '') {
```

Comment:
if we define prop like `protected static string $apiKey;`, ex. is not needed with typed property :)